src/blueprints/timeline-state-resolver-types/atem-types.ts

```diff
@@ -24,3 +35,26 @@ export enum AtemType {
   AUDIO_CHANNEL = 'audioChan',
   MACRO_PLAYER = 'macroPlayer',
 }
+
+export enum AtemTransition {
+  MIX = 0,
+  DIP = 1,
+  WIPE = 2,
+  DVE = 3,
+  STING = 4,
+  CUT = 5,
+  DUMMY = 6
+}
+
+export interface AtemTransitionSettings {
+  mix?: {
+    rate: number
+  }
+  dip?: {
+    rate: number,
+    input: number
+  },
+  wipe?: {
+    //...
+  }
+}
```

Comment:
No (or it wouldn't make logical sense).
These refer to the different MixEffects that Atem is able to do when changing source.
It can't both do a Mix and a Dip effect at the same time.

Reply:
It would be nice to have these as 3 different transition interfaces then unioned as `AtemTransitionSettings`.

Reply:
I agree, but I think we should make those when we start moving stuff to blueprints.